api/proxy/docs/help_out.txt

```diff
@@ -172,6 +172,9 @@ GLOBAL OPTIONS:
     --eigenda.v2.eth-rpc value                                             ($EIGENDA_PROXY_EIGENDA_V2_ETH_RPC)
           URL of the Ethereum RPC endpoint.
    
+    --eigenda.v2.eth-rpc-num-retry value (default: 2)                       ($EIGENDA_PROXY_EIGENDA_V2_ETH_RPC_NUM_RETRY)
+          Number of retries for the Ethereum RPC request after the initial call fails.
```

Comment:
can we further specify the retry policy? is there eg an exponential backoff between attempts or an artificial delay imposed?